testsuite/tests/typing-local/alloc.ml

```diff
@@ -1,5 +1,11 @@
 (* TEST
-   flags += "-extension local" *)
+   * stack-allocation
+   ** native
+      reference = "${test_source_directory}/alloc.stack.reference"
+   * no-stack-allocation
+   ** native
```

Comment:
I think this should be no-stack-allocation + no-flat-floatarray, since it has specific alloc counts.